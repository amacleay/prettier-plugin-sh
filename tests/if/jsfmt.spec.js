run_spec(__dirname, ["sh"]);
