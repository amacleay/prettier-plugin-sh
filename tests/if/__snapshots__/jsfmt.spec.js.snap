// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`if.sh 1`] = `
if true
then
  echo "True"
else
  echo "False"
fi

if true; then
   echo "True"
else
      echo "False"
fi

if true; then
echo "True"
fi
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
if true; then
  echo "True"
else
  echo "False"
fi

if true; then
  echo "True"
else
  echo "False"
fi

if true; then
  echo "True"
fi
`;
